.github/workflows/nightlyfuzz.yml

```diff
@@ -0,0 +1,53 @@
+name: 'nightly/tag fuzz'
+on:
+  schedule:
+    # Note: The schedule event can be delayed during periods of high
+    # loads of GitHub Actions workflow runs. High load times include
+    # the start of every hour. To decrease the chance of delay,
+    # schedule your workflow to run at a different time of the hour.
+    - cron: "25 0 * * *" # at 25 past midnight every day
+  push:
+    tags:
+      - '*'
+  workflow_dispatch: null
+jobs:
+  fuzzrun:
+    name: "run native fuzzers"
+    runs-on: "ubuntu20.04-4cores-16GB"
+    steps:
+      - name: "Checkout"
+        uses: "actions/checkout@v3"
```

Comment:
could you please tag these with the commit sha like this?

`actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1`